CONTRIBUTING.md

```diff
@@ -32,6 +32,42 @@ To get started we recommend:
 
 6. Install [golangci-lint] linter and run `make lint` to execute additional code linters.
 
+### Quick Start Local Development (MacOS)
+
+First install [colima](https://github.com/abiosoft/colima):
+
+```
+brew install colima
+```
+
+Start a colima VM with a single node Kubernetes cluster:
+
+```
+colima start --kubernetes
+```
+
+Then spawn a Docker container with a Go toolchain, with access to the Kubernetes cluster:
+
+```
+docker run -it --rm --network host -v $(pwd):/src -v $HOME/.kube:/root/.kube golang:1.19.5-bullseye bash
+```
+
+A quick test (in the Docker container):
+
+```
+make preflight support-bundle
+bin/preflight config/samples/troubleshoot_v1beta2_preflight.yaml
+bin/preflight examples/preflight/sample-preflight.yaml
```

Comment:
I'd say an example of each of the binaries would be good. I'm guessing support-bundle works too?

Reply:
> Personally, I usually use [docker desktop](https://docs.docker.com/desktop/install/mac-install/) and run [k3d](https://k3d.io/v5.4.6/) to create a k8s cluster. I've not used `calima` before.
> 
> As for the dev environment steps you documented, they look quite involving for someone new to the project. Perhaps we can have the documentation be something shorter like
> 
>     * Launch a k8s cluster. We use `calima` or `docker desktop` + `k3d`.
> 
>     * Ensure you can `kubectl` to your cluster
> 
>     * Run troubleshoot like this... (some commands)

@banjoh I'd love to have a version in the docs that's executable/copy paste from "nothing to workable dev env", rather than having to interpret steps and go through some kind of setup more than that.

how we get there though, I'm completely open to feedback :)